test/groups.yaml

```diff
@@ -17,6 +17,7 @@
 # ------------------------------------------------------------------------------
 general:
     - "jenkins"
+    - "spark"
```

Comment:
i would like to see a spark group to test.

Reply:
added